ui/components/multichain/notification-detail-block-explorer-button/notification-detail-block-explorer-button.tsx

```diff
@@ -27,7 +27,7 @@ export const NotificationDetailBlockExplorerButton = ({
 
   const chainIdHex = decimalToHex(chainId);
   const { nativeBlockExplorerUrl } = getNetworkDetailsByChainId(
-    `0x${chainId}` as keyof typeof CHAIN_IDS,
+    `0x${chainIdHex}` as keyof typeof CHAIN_IDS,
```

Comment:
This function was not using correct input.

I blame the type assertions - will fix and remove these type assertions in a future PR.